src/components/ConditionalWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+interface ConditionalWrapperProps {
+  condition?: boolean
+  wrapper: (children: JSX.Element) => JSX.Element
+  children: JSX.Element
+}
+
+export const ConditionalWrapper = ({
```

Comment:
What is the advantage of using this component over classic ternary operator?

Reply:
I'm a bit puzzled by the aesthetic and declarative arguments as this remove a bit of readability (in my point of view).
Everyone knows how ternary operator works, but we have to open `<ConditionalWrapper>` code to understand what is done.
Also there are now two different `children` variables with different level of imbrication in `<AppWrapper>` (the one from AppWrapper and the one from the `wrapper` prop).

But this is nit. I'm not opposed to merge this PR.